src/main/java/com/carhouse/controller/AjaxRestController.java

```diff
@@ -0,0 +1,29 @@
+package com.carhouse.controller;
+
+import com.carhouse.model.stub.CarModelFilter;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class AjaxRestController {
+
+    @GetMapping("/carMake/{carMakeId}/carModels")
+    public List<CarModelFilter> getCarModels(@PathVariable Integer carMakeId) {
+        List<CarModelFilter> listCarModel;
+        if (carMakeId == -1) {
```

Comment:
what if -2? -3?

Reply:
I set a default value for a list car makes in html to -1 and other values will be from database